examples/resizable/index.html

```diff
@@ -0,0 +1,23 @@
+<html>
+    <head>
+        <title>Jabberwock Editor</title>
+        <link rel="shortcut icon" href="#" />
+    </head>
+    <body style="font-family: sans-serif">
+        <menu style="background-color: #999; height:120px; text-align: center; font-size: 24px; color: white">menu</menu>
+        <main>
+            <h1 style="text-align: center;">In page title.</h1>
+            <p style="text-align: center;">Editor : </p>
+            <div style="border: 1px solid #bbb; margin: 10px 15%;">
+                <section id="contentToEdit">
+                    <h1>Content in section to edit</h1>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus venenatis bibendum pretium. Donec quis fringilla orci. In aliquet lobortis tellus, et congue lectus. Sed libero diam, pulvinar ut nisi ut, tincidunt malesuada lorem. Mauris sollicitudin velit vel auctor ullamcorper. Pellentesque non metus varius, vehicula nulla vitae, lobortis tortor. Aenean tincidunt, augue id efficitur finibus, odio mi varius orci, vitae condimentum mi dui in augue. Donec pulvinar neque metus, id feugiat massa porttitor et. Sed fermentum finibus justo, at dictum tellus venenatis vitae. Fusce aliquam erat ut sapien dapibus tempus at eu sapien. In pellentesque nisi in erat elementum, non malesuada purus elementum. Integer sit amet ex eget ex sodales sollicitudin. Donec id lorem ipsum.</p>
+                    <p>Nulla pellentesque velit vel ex iaculis, a blandit quam dictum. Suspendisse id nunc arcu. Nunc aliquet eros nec leo volutpat interdum a scelerisque magna. In in maximus nisl, eu tincidunt odio. In sit amet lorem scelerisque, fermentum metus eget, mattis ipsum. Praesent dolor urna, vulputate sed sagittis ut, egestas eu orci. Nunc feugiat ac turpis vel accumsan. Pellentesque venenatis nec velit in laoreet. Integer semper egestas turpis non pretium. Etiam cursus dictum turpis in vestibulum. Mauris leo mauris, suscipit ac laoreet eget, consectetur non tellus. In eu ipsum eu lorem commodo mollis. Vivamus tortor massa, maximus at sem eu, ultricies tristique odio. Ut sit amet tempus erat. Etiam quis felis quis erat rutrum efficitur. Donec eleifend sagittis sem, id ultricies enim lobortis vel.</p>
```

Comment:
I would rather add our poem in `jabberwocky.xml` rather than random lorem ipsum ^^